src/main/java/org/sagebionetworks/bridge/exporter/dynamo/DynamoHelper.java

```diff
@@ -95,18 +95,31 @@ public StudyInfo getStudyInfo(String studyId) {
         // it's not null. (This should cover both cases where the attribute doesn't exist and cases where the attribute
         // exists and is null.
         StudyInfo.Builder studyInfoBuilder = new StudyInfo.Builder();
-        if (studyItem.get(STUDY_INFO_KEY_DATA_ACCESS_TEAM) != null) {
-            studyInfoBuilder.withDataAccessTeamId(studyItem.getLong(STUDY_INFO_KEY_DATA_ACCESS_TEAM));
-        }
 
-        if (studyItem.get(STUDY_INFO_KEY_PROJECT_ID) != null) {
-            studyInfoBuilder.withSynapseProjectId(studyItem.getString(STUDY_INFO_KEY_PROJECT_ID));
+        // we need to check at first if the study is disabled exporting or not
+        // treat null as false
+        boolean disableExport = false;
+
+        if (studyItem.get(STUDY_DISABLE_EXPORT) != null) {
+            if (studyItem.getInt(STUDY_DISABLE_EXPORT) != 0) {
+                disableExport = true;
+            }
         }
 
-        if (studyItem.get(STUDY_INFO_KEY_USES_CUSTOM_EXPORT_SCHEDULE) != null) {
-            // For some reason, the mapper saves the value as an int, not as a boolean.
-            boolean usesCustomExportSchedule = studyItem.getInt(STUDY_INFO_KEY_USES_CUSTOM_EXPORT_SCHEDULE) != 0;
-            studyInfoBuilder.withUsesCustomExportSchedule(usesCustomExportSchedule);
+        if (!disableExport) {
+            if (studyItem.get(STUDY_INFO_KEY_DATA_ACCESS_TEAM) != null) {
+                studyInfoBuilder.withDataAccessTeamId(studyItem.getLong(STUDY_INFO_KEY_DATA_ACCESS_TEAM));
+            }
```

Comment:
This seems like a kind of a hack. Instead of hiding all the attributes if export is disabled, it seems cleaner to me to make it part of the StudyInfo and update the logic in RecordFilterHelper.